### DISCUSSION TITLE
Modules only clicking once per session

### DISCUSSION THREAD

**Discussion by steazzy** [polybar/polybar#3099] [Q&A] [answered]
I can click polybar modules only one time during session (except tray), then when I trying to click again i'm getting "polybar-<bar name>-<screen name>" in active windows. When I restarting Awesome WM, I can click one more time and can't click till next restart.

https://github.com/polybar/polybar/assets/108119132/c13393ea-aa3e-4bc3-95f1-647024887f5d

…

 Demonstrated in video

**Comment by patrick96** [comment, marked as answer]
See [Known Issues](https://github.com/polybar/polybar/wiki/Known-Issues#mouse-interactions-not-working-in-awesomewm)

**Reply by steazzy** [reply]
Thanks! That worked

**Reply by steazzy** [reply]
But now fullscreen videos are showing with polybar

**Reply by patrick96** [reply]
In that case, you'll need to play around with the [`wm-restack`](https://github.com/polybar/polybar/wiki/Configuration#bar-settings) option to position polybar below other windows.
